src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
-                variant="contained"
-                data-testid="confirmButton"
-                style={{ width: '100%' }}
-                onClick={() =>
-                  updateFunction(
-                    id,
-                    true,
-                    values.statusValue,
-                    values.amountCurrency,
-                    parseFloat(`${values.amount}`),
-                    values.frequencyValue,
-                  )
-                }
-              >
-                {t('Confirm')}
-              </Button>
-            </Box>
-            <Box className={classes.buttonBottom}>
-              <Button
-                variant="contained"
-                style={{ width: '100%' }}
-                data-testid="doNotChangeButton"
-                onClick={() => updateFunction(id, false)}
-              >
-                {"Don't Change"}
-              </Button>
-            </Box>
-            <Box>
-              <IconButton
-                data-testid="goToContactsButton"
-                onClick={() =>
-                  push({
-                    pathname: `/accountLists/[accountListId]/contacts/[contactId]`,
-                    query: { accountListId, contactId: id },
-                  })
-                }
-              >
-                <SearchIcon />
-              </IconButton>
-              <IconButton
-                data-testid="hideButton"
-                onClick={() => hideFunction(id)}
-              >
-                <VisibilityOffIcon />
-              </IconButton>
-            </Box>
-          </Box>
-        </Grid>
-      </Grid>
+      <Formik
+        initialValues={{
+          statusValue: statusValue,
+          pledgeCurrency: amountCurrency,
+          pledgeAmount: amount,
+          pledgeFrequency: frequencyValue,
+        }}
+        validationSchema={appealFormSchema}
+        onSubmit={async (values) => {
+          await onSubmit(values);
+        }}
+      >
+        {({
+          values: {
+            statusValue,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          },
+          handleSubmit,
+          setFieldValue,
+          errors,
+        }): ReactElement => {
+          const modalContact = {
+            id: id,
+            status: statusValue,
+            name: name,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          };
+
+          return (
+            <Form onSubmit={handleSubmit}>
+              <Grid container className={classes.formWrapper}>
+                <Card className={classes.formInner}>
+                  <Grid container>
+                    <Grid item lg={4} md={4} xs={12}>
+                      <Box
+                        display="flex"
+                        p={2}
+                        alignItems="center"
+                        className={classes.left}
+                      >
+                        <Avatar
+                          src=""
```

Comment:
Let's add the avatar here. You should be able to add it to the graphql if you aren't already.